webdriver-spec.html

```diff
@@ -3359,13 +3326,35 @@
  </tr>
  <tr>
   <td>GET</td>
-  <td>/session/{session id}/window/size</td>
+  <td>/session/{session id}/window/dimensions</td>
  </tr>
 </table>
 
-<p>The <dfn>Get Window Size</dfn> <a>command</a>
- returns the size of the operating system window corresponding
+<p>The <dfn>Get Window Dimensions</dfn> <a>command</a>
+ returns the size and position on the screen
+ of the operating system window corresponding
  to the <a>current top-level browsing context</a>.
+ The returned value is a dictionary with the following members:
+
+ <dl>
+  <dt>x
+  <dd><a>screenX</a> attribute of the <a><code>WindowProxy</code></a>
+   for the <a>current top-level browsing context</a>.
```

Comment:
This will always be CSS pixels. Anything related to position from the web platform is CSS pixels by default